app/src/js/drawable/3d/label3d_handler.ts

```diff
@@ -201,18 +201,59 @@ export class Label3DHandler {
           Session.label3dList.selectedLabelIds, -1, -1
         ))
         return true
-      case Key.P_UP:
-      case Key.P_LOW:
-        Session.dispatch(selectLabel3dType(
-          LabelTypeName.PLANE_3D
-        ))
-        return true
       case Key.B_UP:
       case Key.B_LOW:
         Session.dispatch(selectLabel3dType(
           LabelTypeName.BOX_3D
         ))
         return true
+      case Key.G_UP:
+      case Key.G_LOW:
+        if (Session.label3dList.selectedLabel) {
```

Comment:
It is better to make this a separate function. Also, please add tests for the new feature.